src/common/url/search-params.ts

```diff
@@ -19,3 +19,9 @@ export const createSearchParam = (params: Record<string, string>): string => {
   });
   return urlParams.toString();
 };
+
+export const removeSearchParam = (param: string): string => {
```

Comment:
Just realized that this probably doesn't do anything, since I did not pass in `window.location.search`.